contrib/integration-cli-on-swarm/integration-cli-on-swarm.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+# integration-cli-on-swarm.sh: run the integration tests in parall across a Swarm cluster
+# Please refer to README.md for the usage.
```

Comment:
I wonder if we could find a way to write this driver in something other than bash.

Maybe a `docker-compose.yml` would help reduce the size of this script?